embeddings.go

```diff
@@ -52,6 +52,7 @@ const (
 	AdaCodeSearchText
 	BabbageCodeSearchCode
 	BabbageCodeSearchText
+	AdaSimilarityV2
```

Comment:
I wasn't sure how to name this as it breaks the earlier convention of having multiple embedding models for different use cases. Please let me know if there are any alternate suggestions!